README.md

```diff
@@ -1,8 +1,18 @@
 # capnproto-java: Cap'n Proto for Java
 
-[![Build Status](https://travis-ci.org/capnproto/capnproto-java.svg?branch=master)](https://travis-ci.org/capnproto/capnproto-java)
+[![Build Status](https://travis-ci.org/paxel/capnproto-java.svg?branch=master)](https://travis-ci.org/capnproto/capnproto-java)
 
 [Cap'n Proto](http://capnproto.org) is an extremely efficient protocol for sharing data
 and capabilities, and capnproto-java is a pure Java implementation.
 
-[Read more here.](https://dwrensha.github.io/capnproto-java/index.html)
+[Read more in the source of this fork.](https://dwrensha.github.io/capnproto-java/index.html)
+
+Improvements that have been introduced here:
+
+* dropped support for JDK 7 (question -> reply ("because of this.")).apply("how is that an improvement?")
+* toString() - The generated source now has toString() so you can log what you receive or send
```

Comment:
Is the `toString()` output in the [text format](https://capnproto.org/capnp-tool.html) readable by the `capnp decode` tool? That would be really cool, if so.

Reply:
I actually didn't know until a few days ago about that feature. I will look into the format and try to adapt (if possible)